### PR TITLE
add script to set initial roles

### PR DESCRIPTION
**Pull request by attwad**
I didn't go the full typescript/npm package way as Kelly did for the other scripts as this is really a one-liner doable in mongo shell only.

### CODE REVIEW

**Comment by attwad** [comment]
[Script to assign admin role to a particular user](https://trello.com/c/R4unLgaJ/145-script-to-assign-admin-role-to-a-particular-user)

File: verification/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+This directory contains various scripts to manage the verification part of the epid system.
+
+## Bootstrapping admin roles
+
+When the application is first used, logged-in users have no roles associated with them.
+To bootstrap the admin process, you can assign a user with a given email the admin, reader and curator roles:
+
+```shell
+mongo "mongodb://user:pass@some-connection-string/somedb" \
```
Comment (allysonjp715) [inline verification/scripts/README.md:11]: Would "user:pass@some-connection-string/somedb" need to be changed when run? I wouldn't know what to fill into those fields.

Comment (attwad) [reply verification/scripts/README.md:11]: For localhost (dev) this is "mongodb://localhost:27017/covid19" but for the atlas DB you have to use a proper connection string as the data service or curator services are doing on AWS. (you can go to the web atlas console and generate creds for you, they'll also show you the connection string to use those creds you just created)

Comment (allysonjp715) [reply verification/scripts/README.md:11]: Could you update the instructions with that?

Comment (attwad) [reply verification/scripts/README.md:11]: certainly, done

**Pull request merged**